.github/workflows/e2e-auth.yml

```diff
@@ -137,36 +135,57 @@ jobs:
 
       - uses: actions/setup-go@v3
         with:
-          go-version: 1.16
+          go-version: 1.18
+
+      ## Setup a Knative cluster to test the service
+      - name: Create k8s Kind Cluster
+        run: bash ./runner/scripts/01-kind.sh
+
+      - name: Install Serving
+        run: bash ./runner/scripts/02-serving.sh
 
+      - name: Install Kourier
+        run: bash ./runner/scripts/02-kourier.sh
+
+      - name: Setup domain
+        run: |
+          INGRESS_HOST="127.0.0.1"
+          KNATIVE_DOMAIN=$INGRESS_HOST.nip.io
```

Comment:
this is a bug! and it appears elsewhere in this PR

there is no nip.io anymore, we use sslip.io instead
https://knative.dev/docs/install/yaml-install/serving/install-serving-with-yaml/#configure-dns

Reply:
Thanks!!
Good hint